packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+/**
```

Comment:
we can pull this enum as we aren't supporting non-anchor elements and this isn't configurable